src/commands/dev/lib/default-should-serve.ts

```diff
@@ -0,0 +1,23 @@
+import { parse } from 'path';
+// @ts-ignore
+import { ShouldServeParams } from '@now/build-utils';
+
+export default function shouldServe({
+  entrypoint,
+  files,
+  requestPath
+}: ShouldServeParams): boolean {
+  requestPath = requestPath.replace(/\/$/, ''); // sanitize trailing '/'
+  entrypoint = entrypoint.replace(/\\/, '/'); // windows compatibility
+
```

Comment:
This doesn't seem to be considering the scenario where `entrypoint` has been deleted from `files`, and it should return `false` in that case.

Reply:
Didn't know that's possible 😅

When would it occurs? Should we update these description?
https://github.com/zeit/now-builders/blob/945eb24bdc320e5935c4409024882deddae15c4e/packages/now-build-utils/src/types.ts#L23

Reply:
Yes we should update that.